artifactory/commands/transferfiles/localgeneratedfilter.go

```diff
@@ -0,0 +1,136 @@
+package transferfiles
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/jfrog/gofrog/datastructures"
+	"github.com/jfrog/gofrog/version"
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const (
+	localGeneratedApi                      = "api/localgenerated/filter/paths"
+	minArtifactoryVersionForLocalGenerated = "7.55.0"
+)
+
+// The request and response payload of POST '/api/localgenerated/filter/paths'
+type LocalGeneratedPayload struct {
```

Comment:
I suggest renaming "...LocalGenerated..." to "...LocallyGenerated..." all across the codebase (exists in multiple function name and variables).